### ISSUE TITLE
ObjectClass missing 'required' type under ldapEntityType

### ISSUE THREAD

**Issue by ryanesch** [OpenLiberty/open-liberty#19157]
Under ldapRegistry's ldapEntityType, objectClass is required. This is missing from the 'Type' column.
https://openliberty.io/docs/21.0.0.9/reference/config/ldapRegistry.html

**Comment by ryanesch** [comment]
It may also help to add an example to the ldapRegistry 'name' description.
"The name of the LDAP entity type, eg. PersonAccount, Group, or OrgContainer."
These are not the only 3 entity type name options, but they are the ones used 99% of the time.

**Comment by dmuelle** [comment]
Hi @ryanesch -I'm transferring this issue to the main open-liberty repo as  the page you referenced-and all pages in `/config` section of the doc- are part of the autogenerated docs, which are maintained by development. The pages are generated by the build from the metatype files. I believe particular file for this page is `com.ibm.ws.security.wim.adapter.ldap/resources/OSGI-INF/l10n/metatype.properties` but someone from dev can confirm that.

**Comment by ryanesch** [comment]
Yeah, my team owns that file. The objectclass attribute is listed as required, hence my confusion.
<OCD id="com.ibm.ws.security.wim.adapter.ldap.entityType" name="%supportedLdapEntityType" description="%supportedLdapEntityType.desc">
	<AD id="name" name="%ldapEntityTypeName" description="%ldapEntityTypeName.desc"
		required="true" type="String" />
	<AD id="searchFilter" name="%searchFilter" description="%searchFilter.desc"
		required="false" type="String" />
	<AD id="objectClass" name="%objectClass" description="%ldapEntityTypeObjectClass.desc"
		required="true" type="String" cardinality="2147483647" />
	<AD id="searchBase" name="%searchBase" description="%searchBase.desc"
		required="false" type="String" cardinality="2147483647" />

**Comment by dmuelle** [comment]
ok yeah- I see that now in the metatype.xml. That is confusing. There's nothing on the ID end of things that would stop it from showing up as required.

**Comment by ryanesch** [comment]
Okay, thanks David.
